.github/workflows/main.yml

```diff
@@ -42,14 +42,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
       - name: Set up Node.js 6
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
         with:
           node-version: 6
       - name: Install dependencies
         run: npm install
-      - name: Downgrade mocha
+      - name: Install mocha 6
```

Comment:
We are keeping Node.js 6 compatibility because Babel needs it, but @mathiasbynens how do you feel about dropping it once we release Babel 8 (which will drop support for all the currently not maintained Node.js versions)?